app/server/utils/ussd/kenya_ussd_processor.py

```diff
@@ -76,17 +76,45 @@ def custom_display_text(menu: UssdMenu, ussd_session: UssdSession) -> str:
 
         user = ussd_session.user
 
-        if menu.name == 'about_my_business':
+        if menu.name == 'about_me':
```

Comment:
FYI I haven't really reviewed this section, it includes ussd protocol stuff I'm not familiar with